dali/base/dadfs.cpp

```diff
@@ -11999,7 +12022,7 @@ IDFAttributesIterator* CDistributedFileDirectory::getLogicalFilesSorted(
     __int64 *cacheHint,
     unsigned *total)
 {
-    class CDFUPager : public CSimpleInterface, implements IElementsPager
```

Comment:
All the IElementsPagers rely on collecting all the results, whether they get the total from the server as meta info or not.
So I don't think it's worth having a separate interface which separates out the capability, i.e. getTotalElements() could be implemented cheaply in all the implementations.
Or you could add a throwUnimplemented() in other implementations of IElementsPager:: if tricky (but I think not) as none are called.